docs/modules/usage/llms/google-llms.md

```diff
@@ -1,28 +1,43 @@
 # Google Gemini/Vertex LLM
 
-## Completion
-
 OpenHands uses LiteLLM for completion calls. The following resources are relevant for using OpenHands with Google's LLMs:
 
 - [Gemini - Google AI Studio](https://docs.litellm.ai/docs/providers/gemini)
 - [VertexAI - Google Cloud Platform](https://docs.litellm.ai/docs/providers/vertex)
 
-### Gemini - Google AI Studio Configs
+## Gemini - Google AI Studio Configs
 
-To use Gemini through Google AI Studio when running the OpenHands Docker image, you'll need to set the following environment variables using `-e`:
+To use Gemini through Google AI Studio when running OpenHands, you'll need to set the following environment variable
+using `-e` in the [docker run command](/modules/usage/getting-started#installation):
 
 ```
 GEMINI_API_KEY="<your-google-api-key>"
```

Comment:
This is not accurate actually, the user doesn't need to have a specially named gemini key in the env. They can use the api key we send to liteLLM the regular way.

(this should be the case with all others, because for liteLLM it's the same thing: it uses the key either from a function parameter if we send it, either it looks it up in the environment)

Reply:
Oh really? That would make it more simple.

Reply:
I'm hoping to be able to run through these docs at some point and make them really accurate.

Reply:
Yes, I just tried Gemini, and I definitely never added an extra env var.

Reply:
Updated.